tests/CssToInlineStylesTest.php

```diff
@@ -1,241 +0,0 @@
-class CssToInlineStylesTest extends \PHPUnit_Framework_TestCase
-{
-    /**
-     * @var CssToInlineStyles
-     */
-    protected $cssToInlineStyles;
-
-    public function setUp()
-    {
-        $this->cssToInlineStyles = new CssToInlineStyles();
-    }
-
-    public function teardown()
-    {
-        $this->cssToInlineStyles = null;
-    }
-
-    public function testSimpleElementSelector()
-    {
-        $html = '<div></div>';
-        $css = 'div { display: none; }';
-        $expected = '<div style="display: none;"></div>';
-        $this->runHTMLToCSS($html, $css, $expected);
-    }
-
-    public function testSimpleCssSelector()
-    {
-        $html = '<a class="test-class">nodeContent</a>';
-        $css = '.test-class { background-color: #aaa; text-decoration: none; }';
-        $expected = '<a class="test-class" style="background-color: #aaa; text-decoration: none;">nodeContent</a>';
-        $this->runHTMLToCSS($html, $css, $expected);
-    }
-
-    public function testSimpleIdSelector()
-    {
-        $html = '<img id="IMG1">';
-        $css = '#IMG1 { border: 1px solid red; }';
-        $expected = '<img id="IMG1" style="border: 1px solid red;">';
-        $this->runHTMLToCSS($html, $css, $expected);
-    }
-
-    public function testInlineStylesBlock()
-    {
-        $html = <<<EOF
-<style type="text/css">
-  a {
-    padding: 10px;
-    margin: 0;
-  }
-</style>
-<a></a>
-EOF;
-        $expected = '<a style="margin: 0; padding: 10px;"></a>';
-        $this->cssToInlineStyles->setUseInlineStylesBlock();
-        $this->cssToInlineStyles->setHTML($html);
-        $actual = $this->findAndSaveNode($this->cssToInlineStyles->convert(), '//a');
-        $this->assertEquals($expected, $actual);
-    }
-
-    public function testStripOriginalStyleTags()
-    {
-        $html = <<<EOF
-<style type="text/css">
-  a {
-    padding: 10px;
-    margin: 0;
-  }
-</style>
-<a></a>
-EOF;
-        $expected = '<a></a>';
-        $this->cssToInlineStyles->setStripOriginalStyleTags();
-        $this->cssToInlineStyles->setHTML($html);
-        $actual = $this->findAndSaveNode($this->cssToInlineStyles->convert(), '//a');
-        $this->assertEquals($expected, $actual);
-
-        $this->assertNull($this->findAndSaveNode($actual, '//style'));
-    }
-
-    public function testSpecificity()
-    {
-        $html = '<a class="one" id="ONE"><img class="two" id="TWO"></a>';
-        $css = <<<EOF
-a {
-  border: 1px solid red;
-  padding: 10px;
-  margin: 20px;
-  width: 10px !important;
-}
-.one {
-  padding: 15px;
-  width: 20px !important;
-}
-#ONE {
-  margin: 10px;
-  width: 30px;
-}
-img {
-  border: 2px solid green;
-}
-a img {
-  border: none;
-}
-EOF;
-        $expected = '<a class="one" id="ONE" style="border: 1px solid red; margin: 10px; padding: 15px; width: 20px !important;"><img class="two" id="TWO" style="border: none;"></a>';
-        $this->runHTMLToCSS($html, $css, $expected);
-    }
-
-    public function testMergeOriginalStyles()
```

Comment:
@tijsverkoyen this test would not pass anymore in the refactoring. this is the issue I reported in the discussion in https://github.com/tijsverkoyen/CssToInlineStyles/issues/112#issuecomment-159269242

I suggest you to revert the removal of this test, simply updating it to run with the new version of the library (which means mostly updating ``runHTMLToCSS``).
this would allow to catch any regression